docker-compose.yml

```diff
@@ -25,7 +25,6 @@ services:
           - ./backend:/foodies/backend
         ports:
           - "8000:8000"
-        command: bash -c "yarn install && yarn start"
         depends_on: 
           - db
         links: 
```

Comment:
ALso in case you're wondering, in our docker-compose file we specify the environment vars here 
![image](https://user-images.githubusercontent.com/49849754/104106070-b5321700-5267-11eb-8cb5-51e76335e7ca.png)
When we build the docker image, one of the steps in our Dockerfile says `COPY . .` so we still end up copying the dotenv file. And the dotenv file also has the environment var `DB_HOST`. However, the environment var we specified in the compose file take precedence over the ones specified in the `dotenv` file.

Source: https://vsupalov.com/override-docker-compose-dot-env/